src/journal/models.py

```diff
@@ -494,6 +495,7 @@ class Issue(AbstractLastModifiedModel):
     volume = models.IntegerField(default=1)
     issue = models.CharField(max_length=255, default="1")
     issue_title = models.CharField(blank=True, max_length=300)
+    display_title = models.CharField(blank=True, max_length=300)
```

Comment:
Should probably set this as editable=False, since user input will always be overriden by .save (with a comment)